microsoft-365/compliance/customer-key-tenant-level.md

```diff
@@ -28,6 +28,7 @@ Using keys you provide, you can create a data encryption policy (DEP) and assign
 - Teams chat suggestions by Cortana
 - Teams status messages
 - User and signal information for Exchange Online
+- Exchange Online mailboxes that are not already encrypted Customer Key DEPs at the service level.
```

Comment:
Brijesh, please review the change I made here.
I removed 
- Exchange Online
and added this line.